gazebo_plugins/include/gazebo_plugins/gazebo_ros_force.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2012 Open Source Robotics Foundation, Inc.
```

Comment:
Copyright year

Reply:
Are you referring to the lack of an end year, or to it being different from 2018? I think even after the refactoring, there's still enough left from the original plugin to keep the same initial year for each file.

Regarding the lack of a final year, take a look at [this conversation](https://bitbucket.org/osrf/gazebo/issues/2126/man-pages-tests-failing-since-we-entered#comment-33345563).

Reply:
Oh cool, thought end year was needed